services-camel/src/main/java/edu/unc/lib/cdr/routing/MetaServicesRouter.java

```diff
@@ -60,7 +60,7 @@ public void configure() throws Exception {
 			.to("fcrepo:{{fcrepo.baseUrl}}?preferInclude=ServerManaged&accept=text/turtle")
 			.process(mdProcessor)
 			.multicast()
-				.to("direct-vm:imageEnhancements","direct-vm:extractFulltext");
+				.to("direct-vm:imageEnhancements","direct-vm:extractFulltext", "direct-vm:replication");
```

Comment:
This won't replicate FITS extracts since it only applies to original_files, you may need to break replication into a separate route from the original binary processing.  I'd suggest kicking it off after indexing and before enhancements.

I'm also realizing that MODS files may get processed as an original file...  trying to decide if that's an issue.  Doesn't need to be addressed right now though.